lib/workers/global/index.ts

```diff
@@ -31,9 +32,10 @@ export async function getRepositoryConfig(
     globalConfig,
     is.string(repository) ? { repository } : repository
   );
+  const { platform } = GlobalConfig.get();
```

Comment:
```suggestion
  const platform = GlobalConfig.get('platform');
```
is probably better for null safety check.